src/App.js

```diff
@@ -128,7 +97,15 @@ const App = () => {
             7-day moving average of COVID-19 <br />deaths per million people
           </div>
         </h1>
-        <div id='chart'></div>
+        {
+          [geolocationStates.loaded, geolocationStates.disallowed].includes(geolocationState)
+          ? (
+            <div id='chart'></div>
+          )
+          : (
+            <Fallback geolocationState={geolocationState} />
```

Comment:
When the page loads with `geolocationStates.disallowed`, it kinda explodes

Reply:
Hadn't thought it out well. Should make more sense now.